.circleci/config.yml

```diff
@@ -128,14 +128,14 @@ jobs:
       - run:
           name: Install modules and dependencies.
           command: |
-            npm install
+            npm install --unsafe-perm
```

Comment:
Is this change still needed now that we have https://github.com/googleapis/nodejs-logging-winston/pull/25 landed.